twoliter/embedded/Makefile.toml

```diff
@@ -807,11 +807,19 @@ fi
 # because we build host tools with cargo as well, like buildsys and pubsys.
 export CARGO_TARGET_DIR="${BUILDSYS_ROOT_DIR}/target/${BUILDSYS_ARCH}"
 
+# The workspace manifest directory could be under variants, or at the root.
+for ws in variants .; do
```

Comment:
Why loop?

Reply:
> Why loop?

To iterate over both the `variants` and `.` directories. (I didn't mean for this to be quite so obfuscated!)

Reply:
I keep missing that dot!